software/contrib/gate_phaser.md

```diff
@@ -0,0 +1,64 @@
+# Gate Phaser
+
+author: Nik Ansell (github.com/gamecat69)
+
+date: May 2024
+
+labels: sequencer, gates
+
+A script which attempts to answer the question "What would Steve Reich do if he had a EuroPi?".
+
+Gates are sent from outputs 1-6 which are offset (or out of phase) with each other. This creates a group of gates that drift in and out of phase with each other over time.
+
+You can use this script to create Piano Phase type patches, dynamic rhythms which evolve over time and eventually resolve back in phase, or ... well lots of other things that would benefit from having gates which are delayed from each other.
+
+# Inputs, Outputs and Controls
+
+![Operating Diagram](./gate_phaser-docs/gate_phaser.png)
+  
+# Getting started
+
+1. Patch anything you want to trigger with a gate to any of the outputs, for example
+percussions elements, emvelopes, sequencer clocks or samples.
+2. Set the Cycle time in milliseconds using knob 1
+3. Set delay time in milliseconds using knob 2
+4. Set the desired gate delay interval using button 1
+5. Use button 2 to change the behaviour of knob 2 to set the desired gate delay time
+
+# So what is this script actually doing?
+
+**Cycle time** is the time in milliseconds between gate outputs at output 1 when the gate delay multiple for output 1 is set to 0.
```

Comment:
This sentence is really confusing to me.  What happens when the delay multiple is not zero?